karma.dashboard-config.cjs

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+/**
+ * Internal dependencies
+ */
+const {
+  storiesDashboard: webpackConfig,
```

Comment:
I couldn't find what you're talking about

Reply:
Right. That's because I haven't merged it yet :) But it's actually very important for us to collect coverage. Could you pls cherrypick the `webpack.config.test.cjs` from https://github.com/google/web-stories-wp/pull/2583 and use it instead? It will do what you want, just slightly differently.